extensions/blocks/story/player/components/icons.scss

```diff
@@ -0,0 +1,121 @@
+@font-face {
+	font-family: 'Material Icons for WP Stories';
```

Comment:
I have zero context here but Have you considered simply including SVGs instead of a font?

SVG would be easier to maintain and better for web performance since it can be bundled and even shipped incrementally, vs everything at once.

Reply:
I did consider it but I'm not using react/JSX here so I didn't see an easy way to inline it into the code without changing the build.

A bit of context: anything that lives in `story/player/` right now could (and probably will) be moved to an external service later on as we'll need embeds to support sharing stories (Meaning the story will likely be displayed in an iframe loaded from wordpress.com). So it should be seen as a standalone app in itself whose only dependencies are `preact` and `htm`. I'm leveraging the jetpack build right now but I would like to avoid doing any changes to it.

About JSX vs HTM I'm considering converting the whole app to JSX. I'm a bit hesitant as `htm` can allow users to customize the UI without a building step which is quite nice but which is not a priority for us atm

Reply:
Thanks for the context! It's helpful. :-)

One way to inline svg's by having them in JS files, like so:

https://github.com/WordPress/gutenberg/blob/6e80f44539a03d658935303000aecf313dba9127/packages/icons/src/library/align-left.js

Alternatively this works:

```jsx
import Image from './image.svg';

<img src={ Image } />
```

Reply:
I see but your first approach uses JSX.

The second one could work with htm but it will depend on the packager to either inline the image using base64 or move the assets to the right folder and generate the correct url in place. Since it seemed to me that the jetpack packager is doing the second option I did not go that route.

Reply:
The build tooling could be changed so that you can have inlined svg. You've done quite a bit of work in getting things work in font (which I think isn't favorable for maintainability, flexibility and performance) and same effort could've instead one to build-tooling. Jetpack uses [`calypso-build`](https://github.com/Automattic/wp-calypso/tree/master/packages/calypso-build) but things can also be implemented directly at Jetpack since we extend the webpack config from that tool. Doing changes higher up in the source the better, of course.

Reply:
Maybe we could do something like what we do for the Podcast player block instead?
https://github.com/Automattic/jetpack/blob/06e20fbb4eb098c32bc01631f570735b1f55e5ac/extensions/blocks/podcast-player/style.scss#L431-L438

For more info on this: #15857